generator/templates/schema.gotmpl

```diff
@@ -185,5 +185,40 @@ func ({{.ReceiverName}} *{{ pascalize .Name }}) UnmarshalBinary(b []byte) error
   *{{ .ReceiverName }} = res
   return nil
 }
-{{ end -}}
 
+// MarshalText interface implementation
+func ({{.ReceiverName}} *{{ pascalize .Name }}) MarshalText() ([]byte, error) {
+  if {{ .ReceiverName }} == nil {
+    return nil, nil
+  }
+  b, err := swag.WriteJSON({{ .ReceiverName }})
+  if err != nil {
+    return nil, err
+  }
+  return []byte(fmt.Sprintf("%v", b)), nil
+}
+
+// UnmarshalText interface implementation
+func ({{.ReceiverName}} *{{ pascalize .Name }}) UnmarshalText(text []byte) error {
+  var res {{ pascalize .Name }}
+  fmt.Printf("%s", string(text))
+  *{{ .ReceiverName }} = res
+  return nil;
+}
+
+
+// String interface implementation
+func ({{.ReceiverName}} *{{ pascalize .Name }}) String() string {
+  if {{ .ReceiverName }} == nil {
+    fmt.Errorf("Nil calling object")
```

Comment:
this is not used so it should be removed.

Reply:
As far as I recall , you asked me to implement string() function.

So, I remove both UnmarshalText and string()